src/oneTrust/helpers/oneTrustAssessmentToJson.ts

```diff
@@ -24,13 +24,13 @@ export const oneTrustAssessmentToJson = ({
   let jsonEntry = '';
   // start with an opening bracket
   if (index === 0 || wrap) {
```

Comment:
I see the comment for `wrap` says `Whether to wrap every entry in brackets`, and it looks like we want to do that only when importing to transcend (mimicking a CSV)... but what is the `syncOneTrustAssessmentToDisk` option?

Reply:
When using `syncOneTrustAssessmentToDisk`, the output is `[{},{},{}...]` (i.e., the first item starts with `[` and the last ends with `]). Does that answer your question?

Reply:
No, I actually meant what is the difference between sync to transcend and sync to disk.

Reply:
Oh, sync to disk just means writing the pulled data to a json file.

Reply:
Are they both steps in the import process or are they two unique import options?

Reply:
> pulled data to a json file.

Because this is distinct from the risk files that get uploaded to s3, right? Would we ever want an imported assessment in JSON and not in Transcend? If it is in Transcend, why do we need it in JSON? Just for a historical record?

Reply:
We, as Transcend, would always want to sync to Transcend. However, the cli is also available for customer use, and they may find value in keeping the records in Json as well.